echo/server/dembrane/reply_utils.py

```
@@ -133,8 +143,36 @@ async def generate_reply_for_conversation(
         "description": conversation["project_id"]["context"],
         "get_reply_prompt": conversation["project_id"]["get_reply_prompt"],
+        "get_reply_mode": conversation["project_id"]["get_reply_mode"],
+        "default_conversation_title": conversation["project_id"]["default_conversation_title"],
+        "default_conversation_description": conversation["project_id"]["default_conversation_description"],
+        "default_conversation_transcript_prompt": conversation["project_id"]["default_conversation_transcript_prompt"],
     }
 
+    # Check if we should use summaries for adjacent conversations
+    get_reply_mode = current_project.get("get_reply_mode")
+    use_summaries = get_reply_mode in ["summarize", "brainstorm", "custom"]
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)